portal/styles/footer.scss

```diff
@@ -1,3 +1,13 @@
-.email-wrap {
+.contact-details {
   word-break: break-word;
+  margin-block-start: 1em;
+  margin-block-end: 1em;
```

Comment:
This class looks like it's only used on <dl> elements and these are the default values for the <dl> element anyway, so these two lines probably aren't needed.

Reply:
I removed margin from `portal` and `gef` services.